grommet/grommet
# [FileInput] Add onChange callback to remove actions

#### What does this PR do?

Adds `onChange` callback to remove actions

#### Where should the reviewer start?

The FileInput component

#### What testing has been done on this PR?

Manually checking if the `onChange` is being properly executed

#### How should this be manually tested?

Importing the component, pass an `onChange` prop, adds an item, and then remove it. The `onChange` callback should be dispatched on both add and remove actions.

#### Any background context you want to provide?

#### What are the relevant issues?

#5023

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?

No.

#### Should this PR be mentioned in the release notes?

That would be good.

#### Is this change backwards compatible or is it a breaking change?

It depends. Not necessarily breaks, but might lead the user to unexpected behavior.
